packages/system-data/src/fields/policies.yaml

```diff
@@ -0,0 +1,89 @@
+table: directus_policies
+
+fields:
+  - field: id
+    hidden: true
+    special:
+      - uuid
+
+  - field: name
+    required: true
+    width: half
+    display: formatted-value
+    display_options:
+      translate: true
+
+  - field: icon
+    interface: select-icon
+    display: icon
+    width: half
+
+  - field: description
+    interface: input
+    options:
+      placeholder: $t:field_options.directus_policies.description
+    width: full
+    display: formatted-value
+    display_options:
+      translate: true
+
+  - field: permissions
+    interface: system-permissions
+    special:
+      - o2m
+
+  - field: app_access
+    interface: boolean
+    special:
+      - cast-boolean
+    width: half
+
+  - field: admin_access
+    interface: boolean
+    special:
+      - cast-boolean
+    width: half
+
+  - field: ip_access
+    interface: tags
+    options:
+      placeholder: $t:field_options.directus_policies.ip_access
+    special:
+      - cast-csv
+    width: full
+
+  - field: enforce_tfa
+    note: $t:field_options.directus_policies.enforce_tfa
+    interface: boolean
+    special:
+      - cast-boolean
+    width: half
+
+  - field: assigned_to_divider
+    interface: presentation-divider
+    options:
+      title: $t:field_options.directus_policies.assigned_to
+    special:
+      - alias
+      - no-data
+    width: full
+
+  - field: users
+    interface: list-m2m
+    options:
+      template: '{{ user.first_name }} {{ user.last_name }}'
+      junctionFilter:
+        user:
+          _nnull: true
+    special:
+      - m2m
+
+  - field: roles
+    interface: list-m2m
+    options:
+      template: '{{ role.name }}'
+      junctionFilter:
+        role:
+          _nnull: true
```

Comment:
The M2M interface does not show the public role when it's attached.
<img width="863" alt="image" src="https://github.com/directus/directus/assets/26413686/8cb68ca1-d47a-4923-9a94-ecd587065c08">

If the junction filter is removed, the interface will show an empty item for the public role. 🤔 

```suggestion
```

<img width="824" alt="image" src="https://github.com/directus/directus/assets/26413686/f1d4844a-b6ef-4c42-a1c6-18b6a6caccc2">

<img width="834" alt="image" src="https://github.com/directus/directus/assets/26413686/c68a9814-07f2-4c79-b494-1fe5288e4775">

Reply:
Hmm is the public role missing some props?

Reply:
The public role does not exist at all, right? It is a virtual role that is composed of all policies that have `role: null` and `user: null` in `directus_access`

Reply:
So that is to be expected for now. Maybe we can convince the M2M to show this virtual row as well, but not really in the scope of the RC I'd say.